dashboard/app/models/pd/enrollment.rb

```diff
@@ -79,6 +80,12 @@ class Pd::Enrollment < ActiveRecord::Base
     replace_existing
   )
 
+  def set_default_scholarship_info
+    if user && workshop.csf? && workshop.school_year
```

Comment:
Should we do anything special with cases that don't include all three?

Reply:
In practice in the UI, there is no way to create a CSF enrollment without a user or a workshop without a school year (that would mean it has no sessions). In these cases, or for non-csf workshops, we don't create scholarship info by default, so we don't need to do anything else here.